### ISSUE TITLE
Notes are not being displayed in the order that they appear in the text

### ISSUE THREAD

**Issue by cckozie** [unfoldingWord-dev/ts-android#1864]
It looks like they are listed in alphabetical order rather than in the order they appear in the verses.

**Comment by neutrinog** [comment]
also, there's a bug in the words not displaying in psa.

**Comment by neutrinog** [comment]
fixed the words not displaying in psa.
The words are showing in the order given by the api (which is not alphabetical in the examples I checked).
If there's a discrepancy in the word order for a particular item that'll need to get fixed at the source level or at the server level.

**Comment by cckozie** [comment]
They are in the correct order in desktop:
![image](https://cloud.githubusercontent.com/assets/8171759/23278971/12dd4a82-f9d9-11e6-9d54-07f0ee0bff48.png)

Android:
![screenshot_2017-02-23-14-58-22](https://cloud.githubusercontent.com/assets/8171759/23278990/247c431a-f9d9-11e6-8777-5503862b6cd5.png)

**Comment by neutrinog** [comment]
@cckozie make sure you update the source.

**Comment by cckozie** [comment]
OK, new source fixed it.

**Comment by cckozie** [comment]
![screenshot_2017-03-21-09-14-09](https://cloud.githubusercontent.com/assets/8171759/24152496/d0515ab4-0e19-11e7-9997-8c64bc181a86.png)
This was also "fixed" by downloading the latest source. Shouldn't the latest source be bundled with the app?

**Comment by neutrinog** [comment]
yes, we usually do that once we are ready for a release. Otherwise we just have to do it again.
